test/E2E/durableApp/FanOutOnInputOrchestrator/run.ps1

```diff
@@ -0,0 +1,15 @@
+using namespace System.Net
+
+param($Context)
+
+$ErrorActionPreference = 'Stop'
+
+$output = @()
+# call activity in every input
+foreach ($input in $Context.Input)
+{
```

Comment:
Could you please add a comment describing why this tests whether the input type is serialized correctly?

Reply:
Added :)